src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -218,115 +203,64 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                     'const' => self::SPACING_ONE,
                     'method' => self::SPACING_ONE,
                     'property' => self::SPACING_ONE,
-                    'trait_import' => self::SPACING_ONE,
+                    'trait_import' => self::SPACING_NONE,
```

Comment:
why changing the default? we had cases in past when we considered such change as bc-breaker, we were planning change of default configuration of fixer for next major always

Reply:
This was added in the last patch and take as not a default and reported as `bug`, as such patching to new default 
Btw. all (OSS) repo's I checked use this CS, also it replaces the `no_extra_lines` option (to fix the conflict) which also fixes to this as default.

Reply:
This CS is also used in examples in PSR-12.